exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,6 @@
-const name = " Daniel  ";
+const name = "jaspreet";
+const nameLength = name.length;
+
+console.log(`my name is ${name} and its ${nameLength} character long` .trim()); 
```

Comment:
the `.trim()` should be used differently here, e.g. `${name.trim()}`
try updating line 1 to be something like `const name = "     jaspreet      "`